rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -1752,7 +1753,7 @@ namespace rsx
 		registers[NV406E_SET_CONTEXT_DMA_SEMAPHORE] = CELL_GCM_CONTEXT_DMA_SEMAPHORE_R;
 		registers[NV4097_SET_CONTEXT_DMA_SEMAPHORE] = CELL_GCM_CONTEXT_DMA_SEMAPHORE_RW;
 
-		if (get_current_renderer()->isHLE)
+		if (auto rsx = get_current_renderer(); !rsx || rsx->isHLE)
```

Comment:
I think this should run unconditionally. We lose nothing if GCM overwrites our setup anyway.
I fail to see a situation where we would run registers init and not expect that the initial register state matches a PS3.

Reply:
Agreed.